src/main/java/jackbean/task/Event.java

```diff
@@ -0,0 +1,64 @@
+package jackbean.task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents an event task in the JackBean chatbot.
+ * This JavaDoc was written by GitHub Copilot.
+ */
+public class Event extends Task {
+    protected LocalDate from;
```

Comment:
Should the names of these fields be changed to nouns?

Reply:
Sure!